src/frontend/src/pages/Groups/GroupsDataTable.jsx

```diff
@@ -23,15 +25,26 @@ export function GroupsDataTable({ groups }) {
         { key: 'memberCount', header: 'Members' }
     ];
 
+    const navigate = useNavigate();
+
     const getRow = rowId => groups.find(({ id }) => id == rowId);
 
     return (
         <SmsDataTable rows={groups} headers={headers} className='groups-page__datatable'>
             {({ rows, headers, getHeaderProps, getRowProps, getTableProps, onInputChange }) => (
-                <TableContainer title='Groups'>
+                <TableContainer 
+                    title='Groups'
+                    description='This is a list of all the project for this semester.'
```

Comment:
@Ryan-Cleminson see #64 to finalise all of these. I'm sure Wayne will have some minor input on at least one of these descriptions